contracts/test/shared/functions.t.sol

```diff
@@ -107,6 +108,8 @@ contract Functions is Constants, Errors, TestStorage, Signatures {
     if (revertType_ == RevertStatus.Success) {
       assertEq(marketplace.listingPrice(listingId), salePrice);
       assertTrue(marketplace.isListingActive(listingId), 'Functions: isListingActive');
+      assertEq(IERC721(tokenContract).ownerOf(tokenId), address(marketplace));
+      assertEq(IERC721(tokenContract).balanceOf(sender), balance - 1);
```

Comment:
- Let's organize these assertion in a separate function verify_createListing, and reuse it for both helper_createListing
- Add a check balanceOf(marketPlace) + 1